### DISCUSSION TITLE
Use a lambda layer for vendor folder?

### DISCUSSION THREAD

**Discussion by mauriciogior** [brefphp/bref#1702] [Support]
Our vendor folder is almost hitting the 250mb limit, and every deployment takes a long time to zip and upload, but our codebase is quite small.

I was thinking on creating a layer for our vendor folder with preload and only zip the actual code. Is that possible?

**Comment by mnapoli** [comment]
It should be possible, `serverless.yml` allows defining layers. But this is not documented in Bref (it's not a commonly used feature for Bref users), you will have to check out the serverless.com docs.

You can also switch to deploying using Docker.
